pinot-controller/src/main/java/com/linkedin/pinot/controller/helix/core/realtime/PinotLLCRealtimeSegmentManager.java

```diff
@@ -641,8 +641,11 @@ protected SegmentMetadataImpl extractSegmentMetadata(final String rawTableName,
   }
 
   public LLCRealtimeSegmentZKMetadata getRealtimeSegmentZKMetadata(String realtimeTableName, String segmentName) {
```

Comment:
Use @Nullable annotation on return type.
caller needs to handle null return. commitSegment() above will throw NPEs otherwise